lomap/dbmol.py

```diff
@@ -331,13 +306,13 @@ def read_molecule_files(self):
         logging.info(30 * '-')
 
         # The .mol2 and .sdf file formats are the only supported so far
-        mol_fnames = glob.glob(self.options.directory + "/*.mol2")
-        mol_fnames += glob.glob(self.options.directory + "/*.sdf")
+        mol_fnames = glob.glob(self.options['directory'] + "/*.mol2")
```

Comment:
There's an issue here in the way that LOMAP handles molecules: generating networks with the same series of ligands in both .sdf and .mol2 (switching through e.g. obabel) formats results in different graph outputs (i.e. different sets of suggested edges). Anecdotally I've observed that .sdf-based graphs are better (i.e. lead to ultimately more accurate FE predictions) than .mol2-based graphs, likely due to RDKit handling the molecule information in a different way. 

Out of scope for this PR I think but would be good to deal with at some point soon.

Reply:
I think in general having an API based on files (not molecular representations like rdkit) isn't a great idea, but it would be too much of a change to make.

Reply:
Agreed - I don't know about OFE but for direct users of LOMAP at the very least a log warning that discourages the use of .mol2 inputs would be a reasonable idea for the time being.